### PR TITLE
fix(suite-native): track btc firmware instead of btc device to analytics

### LINKED ISSUE #15145: Analytics shoud track bitcoin-only firmware usage, not bitcoin-only devices
In `useReportDeviceConnectToAnalytics`, property `isBitcoinOnly` should track usage of bitcoin-only firmware, not bitcoin-only devices. Regular devices can also have bitcoin-only firmware installed so monitoring this is more useful than tracking orange devices.

Remember to document any changes in the [analytics table](https://www.notion.so/satoshilabs/connect_device-f1106223e5044c04a26029139e9475af).

### PR DESCRIPTION
**Pull request by vytick**
Report BTC only firmware instead BTC only device to analytics

## Related Issue

Resolve #15145

### CODE REVIEW

**Review by PeKne — APPROVED** [review]

File: suite-native/device/src/hooks/useReportDeviceConnectToAnalytics.ts [hunk 1]
```diff
@@ -17,7 +17,7 @@ export const useReportDeviceConnectToAnalytics = () => {
     const deviceFirmwareVersion = useSelector(selectDeviceFirmwareVersion);
     const isDeviceProtectedByPin = useSelector(selectIsDeviceProtectedByPin);
     const deviceModel = useSelector(selectDeviceModel);
-    const isDeviceBitcoinOnly = useSelector(selectIsDeviceBitcoinOnly);
+    const isDeviceBitcoinOnly = useSelector(selectHasBitcoinOnlyFirmware);
```
Comment (PeKne) [inline suite-native/device/src/hooks/useReportDeviceConnectToAnalytics.ts]: nit: 
```suggestion
    const hasBitcoinOnlyFirmware = useSelector(selectHasBitcoinOnlyFirmware);
```

**Comment by matejkriz** [comment]
Please don't forget to mention in changelog and ping @kolomaznikt https://www.notion.so/satoshilabs/Changelog-Mobile-app-946307159a294bcdb90b94e90eefd292?pvs=4

**Comment by vytick** [comment]
> Please don't forget to mention in changelog and ping @kolomaznikt [notion.so/satoshilabs/Changelog-Mobile-app-946307159a294bcdb90b94e90eefd292?pvs=4](https://www.notion.so/satoshilabs/Changelog-Mobile-app-946307159a294bcdb90b94e90eefd292?pvs=4)

Yes, I have discussed with him, how to do that, and will add note to the event.
https://www.notion.so/satoshilabs/connect_device-f1106223e5044c04a26029139e9475af

File: suite-native/device/src/hooks/useReportDeviceConnectToAnalytics.ts (hunk 1 above)
Comment (vytick) [reply suite-native/device/src/hooks/useReportDeviceConnectToAnalytics.ts]: 59d97914e4

**Review by matejkriz — APPROVED** [review]

**Pull request merged**